### DISCUSSION TITLE
如何AutoScroll到指定的field，是否有提供直接的方案？

### DISCUSSION THREAD

**Discussion by dxiongya** [alibaba/formily#3842] [Q&A 2.x]
我在使用formily/antd-v5版本，我大概看了下好像没看到什么方案，有什么解决方案吗

**Comment by Landon-CN** [comment]
是不是希望滚动到校验失败的地方？粗暴的方式就是通过querySelector选中第一个feedback error，然后使用dom.scrollIntoViewIfNeeded()

**Reply by dxiongya** [reply]
哈哈，现在是这样做的，其实看看能不能有更合适的，其实我觉得formily中的antd formitem label这层可以加一个for，这样不提供我们自己写也可以比较明确找到.

**Reply by Landon-CN** [reply]
@chongqiangchen 没太看懂for是啥属性

**Comment by yanhao98** [comment]
useField拿到的field数据有个data, 可以在组件挂载时将组件的el存到这里面。需要的时候就可以找到对应字段的el元素。
